### ISSUE TITLE
ubuntu/5.0 docker build failing due to file copy in Docker

### ISSUE THREAD

**Issue by tomoki10** [aws/aws-codebuild-docker-images#517]
**Describe the bug**
Target file: aws-codebuild-docker-images/ubuntu/standard/5.0/Docker

Dockerfile tries to move a file that does not exist on Docker and fails.
```
tools/runtime_configs/python/$PYTHON_37_VERSION
tools/runtime_configs/python/$PYTHON_38_VERSION
tools/runtime_configs/python/$PYTHON_39_VERSION

tools/runtime_configs/php/$PHP_73_VERSION
tools/runtime_configs/php/$PHP_74_VERSION
tools/runtime_configs/php/$PHP_80_VERSION
```

**To Reproduce**
1. Go to aws-codebuild-docker-images/ubuntu/standard/5.0
2. Run `docker build -t aws/codebuild/standard:5.0 .`
3. I get the following error statement
```
executor failed running [/bin/sh -c cat tools/runtime_configs/python/$PYTHON_37_VERSION > /root/.pyenv/plugins/python-build/share/python-build/$PYTHON_37_VERSION]: exit code: 1
```

The following lines in the target file are relevant.

https://github.com/aws/aws-codebuild-docker-images/blob/f125ad2319d02e7a1478f9b661692f04bf7b7a50/ubuntu/standard/5.0/Dockerfile#L251
Line251, 259, 267, 282, 288, 292

4. Commenting out the above line will result in the following error

```
executor failed running [/bin/sh -c cat ssh_config > /root/.ssh/config]: exit code: 1
```
Files such as ssh_config do not exist on Docker. Revert to the old commit around Line 469 and it works.

Before:
```
RUN cat ssh_config > /root/.ssh/config
RUN cat runtimes.yml > /codebuild/image/config/runtimes.yml
RUN cat dockerd-entrypoint.sh > /usr/local/bin/dockerd-entrypoint.sh
RUN chmod +x /usr/local/bin/dockerd-entrypoint.sh
RUN cat legal/bill_of_material.txt     > /usr/share/doc/bill_of_material.txt
RUN cat amazon-ssm-agent.json          > /etc/amazon/ssm/amazon-ssm-agent.json
```

After:
```
COPY ssh_config /root/.ssh/config
COPY runtimes.yml /codebuild/image/config/runtimes.yml
COPY dockerd-entrypoint.sh /usr/local/bin/
COPY legal/bill_of_material.txt     /usr/share/doc
COPY amazon-ssm-agent.json          /etc/amazon/ssm/
```

**Expected behavior**
Build of image should run without issue.

**Logs**
```
executor failed running [/bin/sh -c cat tools/runtime_configs/python/$PYTHON_37_VERSION > /root/.pyenv/plugins/python-build/share/python-build/$PYTHON_37_VERSION]: exit code: 1
```

**Platform (please complete the following information):**
 - OS: Mac x86

**Additional context**
The following commitments may be relevant.
https://github.com/aws/aws-codebuild-docker-images/commit/f125ad2319d02e7a1478f9b661692f04bf7b7a50

**Comment by hborham** [comment]
Same issue for me

**Comment by hborham** [comment]
@leoherran-aws this seems related to your last commit.

**Comment by leoherran-aws** [comment]
Fixed in 282c6634e8c83c2a9841719b09aabfced3461981. Thanks for letting us know about this issue.
